sdk/servicebus/service-bus/sample.env

```diff
@@ -0,0 +1,26 @@
+# Used in most samples. Retrieve these values from a storage account in the Azure Portal.
+SERVICE_BUS_ENDPOINT="<service bus namespace>.servicebus.windows.net"
+SERVICE_BUS_CONNECTION_STRING="<connection string>"
```

Comment:
Should be `SERVICEBUS_CONNECTION_STRING` for the tests.

The samples should probably be fixed. It appears they're using `SERVICE_BUS_CONNECTION_STRING` (extra underscore).

Reply:
I wonder, is this just for samples? I'll investigate. @willmtemple should we move the samples code to use these parent level sample.env instead of custom sample.env files in the sample folders?

Reply:
I'll follow up on this later today.

Reply:
So, instead of making a follow-up PR, I'll make an issue!

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/11877